templates/education/programs.php

```diff
@@ -66,20 +71,21 @@
 			<div class="boxey-inner">
 
 				<div class="iso-title">
-					<?php the_title(); ?>
+					<?php echo esc_html( $the_title ); ?>
 				</div>
 
 				<div class="iso-description hidden">
 					<?php the_content(); ?>
-					<?php echo esc_html( $department ); ?>
-					<?php echo esc_html( $location ); ?>
+					<?php echo esc_attr( $department ); ?>
+					<?php echo esc_attr( $location ); ?>
+					<?php echo esc_html( $the_title ); ?>
 				</div>
 
 				<div class="excerpt">
 
 					<div class="location">
 						<?php echo esc_html( $location ); ?>
-						<small><?php echo esc_html( $department ); ?></small>
+						<small><?php echo sanitize_text_field( $department ); ?></small>
```

Comment:
Sanitization is for input, not output. `wp_kses_post()` may be more appropriate?